lib/browser/api/browser-view.ts

```diff
@@ -105,12 +114,18 @@ export default class BrowserView {
       newViewBounds.width = newBounds.width / this.#autoHorizontalProportion.width;
       newViewBounds.x = newBounds.width / this.#autoHorizontalProportion.left;
     }
+
     if (this.#autoVerticalProportion) {
       newViewBounds.height = newBounds.height / this.#autoVerticalProportion.height;
       newViewBounds.y = newBounds.y / this.#autoVerticalProportion.top;
     }
+
     if (this.#autoHorizontalProportion || this.#autoVerticalProportion) {
-      this.#webContentsView.setBounds(newViewBounds);
+      this.#webContentsView.setBounds(
+        Object.entries(newViewBounds).reduce((acc, [key, value]) => ({
+          ...acc, [key]: Math.round(value)
+        }), {} as Rectangle)
```

Comment:
whoa there cowpoke,
```suggestion
        {
          x: Math.round(newViewBounds.x),
          y: Math.round(newViewBounds.y),
          width: Math.round(newViewBounds.width),
          height: Math.round(newViewBounds.height),
        }
```

but perhaps we should auto-round in the Rectangle converter on the C++ side?

Reply:
> but perhaps we should auto-round in the Rectangle converter on the C++ side? 

this might be cleaner, I agree! I'll try that.